hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1063,7 +1045,269 @@ public function createInsurancePlanTypes() {
     }
   }
   
-  function decToFraction($fte) {
+  /**
+   * Upgrader to :
+   * 
+   * - Remove unused pay scales except 'Not Applicable'
+   * - Remove unused Hour Locations except 'Head Office'
+   * - Remove Duplicated 'Employee - Permanent' Contract Type
+   * - Add 'Fixed Term' Contract Type
+   * - Sort contract types alphabetically
+   * - Add 'Retirement' contract end reason
+   * - Add new contract change reason options
+   *
+   * @return TRUE
+   */
+  public function upgrade_1026() {
+    $this->up1026_removeUnneededPayScales();
+    $this->up1026_removeUnneededHourLocations();
+    $this->up1026_removeDuplicateContractType();
+    $this->up1026_removeRevisionChangeReasons();
+
+    $optionValues = [
+      'hrjc_contract_type' => ['Fixed Term'],
```

Comment:
oh, I meant add instead of remove. As you mentioned yourself, these new options are not on the files and need to be added. 

Sorry for the confusion